cmd/installer/cli/install.go

```diff
@@ -96,15 +96,24 @@ func InstallCmd(ctx context.Context, name string) *cobra.Command {
 			cancel() // Cancel context when command completes
 		},
 		RunE: func(cmd *cobra.Command, args []string) error {
-			if err := verifyAndPrompt(ctx, name, flags, prompts.New()); err != nil {
+			var airgapInfo *kotsv1beta1.Airgap
+			if flags.airgapBundle != "" {
+				var err error
+				airgapInfo, err = airgap.AirgapInfoFromPath(flags.airgapBundle)
+				if err != nil {
+					return fmt.Errorf("failed to get airgap info: %w", err)
+				}
+			}
```

Comment:
Is there any reason this is not in the preRunInstall function as [suggested](https://github.com/replicatedhq/embedded-cluster/pull/2336#discussion_r2157646365)? That way it exists with the rest of the flag parsing code and will work for both install and install run-preflights command.

Reply:
I'll create a new flag like the rest